source/particle/interpolator/bilinear_least_squares.cc

```diff
@@ -201,24 +286,15 @@ namespace aspect
             {
               prm.enter_subsection("Bilinear least squares");
               {
-                prm.declare_entry ("Global particle property maximum",
-                                   boost::lexical_cast<std::string>(std::numeric_limits<double>::max()),
-                                   Patterns::List(Patterns::Double ()),
-                                   "The maximum global particle property values that will be used as a "
-                                   "limiter for the bilinear least squares interpolation. The number of the input "
-                                   "'Global particle property maximum' values separated by ',' has to be "
-                                   "the same as the number of particle properties.");
-                prm.declare_entry ("Global particle property minimum",
-                                   boost::lexical_cast<std::string>(std::numeric_limits<double>::lowest()),
-                                   Patterns::List(Patterns::Double ()),
-                                   "The minimum global particle property that will be used as a "
-                                   "limiter for the bilinear least squares interpolation. The number of the input "
-                                   "'Global particle property minimum' values separated by ',' has to be "
-                                   "the same as the number of particle properties.");
-                prm.declare_entry("Use limiter", "false",
-                                  Patterns::Bool (),
-                                  "Whether to apply a global particle property limiting scheme to the interpolated "
-                                  "particle properties.");
+                prm.declare_entry("Use linear least squares limiter", "false",
+                                  Patterns::List(Patterns::Bool()),
+                                  "Limit the interpolation of particle properties "
+                                  "onto the cell so the value of each property is no "
+                                  "smaller than its minimum and no larger than its "
+                                  "maximum on the particles in each cell. Currently "
+                                  "doesn't work on spherical grids. If more than one "
```

Comment:
You can remove the limitation of the spherical grids now. Since now all the operations you do are in the reference coordinate system instead of real space it should work independently of the geometry model.

Reply:
This comment is unresolved? Remove the sentence about spherical grids.